Makefile

```diff
@@ -236,11 +236,15 @@ darwin_arm64_notarized: darwin_arm64_signed
 
 windows_install: $(GOTMP)/bin/windows_amd64/ddev_windows_installer.exe
 
-$(GOTMP)/bin/windows_amd64/ddev_windows_installer.exe: $(GOTMP)/bin/windows_amd64/ddev.exe $(GOTMP)/bin/windows_amd64/sudo_license.txt $(GOTMP)/bin/windows_amd64/mkcert.exe $(GOTMP)/bin/windows_amd64/mkcert_license.txt winpkg/ddev.nsi
+windows_sign_binaries: $(GOTMP)/bin/windows_amd64/ddev.exe $(GOTMP)/bin/windows_amd64/mkcert.exe $(GOTMP)/bin/windows_arm64/ddev.exe $(GOTMP)/bin/windows_arm64/mkcert.exe
```

Comment:
I think it makes sense to just have a separate build step `windows_sign_binaries` that applies to both `amd64` and `arm64`, and then the `windows_install` step depends on it but builds the installer only for `amd64` for now. WDYT?

Reply:
It's OK with me. We have one machine (tb-win11-06) that does the signing, has the dongle with the key. I think it can sign arm64 binaries, but you'll need to inspect it on your machine I guess.